Modules/System/Cryptography Management/src/CryptographyManagementImpl.Codeunit.al

```diff
@@ -610,4 +609,90 @@ codeunit 1279 "Cryptography Management Impl."
             Error(GetLastErrorText());
         Hash := Convert.ToBase64String(ByteArray);
     end;
+
+    procedure EncryptTextWithDESCryptoServiceProvider(VarInput: Text; Password: Text; Salt: Text) VarOutput: Text
```

Comment:
May I ask you to move the implementation into separate codeunit within the same module. There is nothing wrong with your implementation here, it is because "Cryptography Management Impl." is getting overcrowded, so it should be easier for maintaining going forward.